locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -1740,28 +1733,28 @@ msgstr ""
 
 msgctxt "SERP footer content"
 msgid "How We Are Profitable"
-msgstr ""
+msgstr "Hogy vagyunk mégis jövedelmezőek"
 
 #. Title for the stuffs explaining <em>how to change your passphrase</em>.
-#. From <a href="https://duckduckgo.com/settings.html">settings page</a>. 
+#. From <a href="https://duckduckgo.com/settings.html">settings page</a>.
 #. You have to click the "What is this?" link to display it.
 msgctxt "cloudsave"
 msgid "How do I change my passphrase?"
-msgstr "Hogyan változtathatom meg a kódszavam?"
+msgstr "Hogyan változtathatom meg a jelmondatom?"
```

Comment:
@zoli111 I just wanted to double check with you the use of "jelmondatom" here and in similar contexts. Automatic translators give me back "motto" rather than "passphrase" or "password" or "codeword", which might be confusing. But I don't know the language, so I might be wrong. Please let me know. 😉